### PR TITLE
Add prop for card style

### PR DESCRIPTION
**Pull request by shem8**
I had a case that I needed to customize the card style (in my case I needed to give the container `width: '100%'` so my view will take the whole space). So I think it'll be nice to add an option for this.

### CODE REVIEW

File: CardStack.js [hunk 1]
```diff
@@ -415,19 +415,19 @@ class CardStack extends Component {
               { translateY: (topCard === 'cardB') ? drag.y : 0 },
               { scale: (topCard === 'cardB') ? 1 : scale },
             ]
-          }}>
+          }, this.props.cardStyle]}>
```
Comment (fudyartanto) [inline CardStack.js]: is it better using `cardContainerStyle` name instead of `cardStyle`?

**Comment by fudyartanto** [comment]
@lhandel I need this one also. Please consider merging this PR.

File: CardStack.js (hunk 1 above)
Comment (shem8) [reply CardStack.js]: agree. renamed.

**Comment by AnatolyMakarovKHM** [comment]
cardContainerStyle: PropTypes.oneOfType([PropTypes.number, PropTypes.object, PropTypes.array]),

for which reason style object should receive PropTypes.number?

**Pull request merged**